packages/neuron-wallet/src/store/WalletStore.ts

```diff
@@ -38,8 +38,14 @@ export default class WalletStore {
     return this.walletIDStore.get(WalletIDKey, [])
   }
 
-  private setIDList = (list: string[]) => {
-    this.walletIDStore.set(WalletIDKey, list)
+  private addIDList = (id: string) => {
```

Comment:
Perhaps `addWalletID` and `removeWalletID`? It's not adding/deleting ID list, but ID (single entry).

Reply:
ok